ibc/doc.go

```diff
@@ -0,0 +1,7 @@
+// Package ibc provides chain and relayer agnostic types and interfaces for the IBC spec.
+// It attempts to expose the domain model of IBC.
+//
+// The official spec documentation can be found at https://github.com/cosmos/ibc/tree/master/spec.
+//
+// Currently, the interfaces may be biased towards chains built with the cosmos sdk and the cosmos/relayer.
+package ibc
```

Comment:
Most of my reasoning is here. I like how this package is our opportunity to exose domain entities and their relationships.

Reply:
+1